### ISSUE TITLE
About default parameters and their ranges

### ISSUE THREAD

**Issue by St-ZFeng** [ComputationalPsychiatry/pyhgf#247]
I've noticed a huge change in this repository. I used it to try to fit a three-level binary HGF model. But I'm a bit confused if it works the way I understand it. In particular, I noticed that the initial parameters of the three-level model seem to be different from those in the MATLAB toolbox. In MATLAB, the initial mean of the second level is fixed to 0 (in the absence of bias), while the initial mean of the first level is determined by the second level via a sigmoid function, and therefore does not need to be defined. The initial mean of the third layer is usually fixed to 1.

![image](https://github.com/user-attachments/assets/f59b02c4-1078-40e4-8ec3-565041c8e21e)

However, I found that in the example provided, these parameters are defined in a different way:

```
three_levels_hgf = HGF(
    n_levels=3,
    model_type="binary",
    initial_mean={"1": 0.0, "2": 0.5, "3": 0.0},
    initial_precision={"1": 0.0, "2": 1.0, "3": 1.0},
    tonic_volatility={"1": None, "2": -3.0, "3": -2.0},
    tonic_drift={"1": None, "2": 0.0, "3": 0.0},
    volatility_coupling={"1": None, "2": 1.0},
    eta0=0.0,
    eta1=1.0,
    binary_precision=jnp.inf,
)
```
`initial_mean={"1": 0.0, "2": 0.5, "3": 0.0}` Should I keep this setting? Or does the initial mean of the second level get converted to a range of 0-1, so 0.5 represents no bias?

Thank you for any help you can provide.

**Comment by LegrandNico** [comment]
Thank you for pointing this out @St-ZFeng . Yes this is an error in the parametrization of the binary HGF in some tests and examples, we should read `0.5` at the first level and `0.0` at the second level. Actually, only the value assigned to the second level matters here as the prediction will be propagated downstream at the first trial and will override the initial mean at the first level.

We have recently removed the input nodes, which greatly simplifies the framework but requires several changes, this was the last major update and we expect more stability going forward now, but let me know if some of your models are not fitting since v0.2.0.

We also need a page that documents the role and range of the parameters, as you mentioned.

TODO:

- [ ] fix the error in parametrization of the binary HGF
- [ ] add a table documenting the parameters' ranges and functions

**Comment by St-ZFeng** [comment]
Looking forward to further updates
